executors/node/segmenter.js

```diff
@@ -0,0 +1,57 @@
+// Tests Intl segmenter
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter
+
+
+module.exports = {
+  testSegmenter: function (json) {
+    const label = json['label'];
+    const locale = json['locale'];
+    let options;
+    let test_option;
+    if (json['options']) {
+      options = json['options'];
+      let desired_granularity;
```

Comment:
suggestion: rename `desired_granularity` to either
* `ecma_granularity`
* `intl_granularity`
* `intl_api_granularity`
* `js_segmenter_granularity`

"desired" is not as clear and informative as it could be about how it's being used.

Reply:
Clarified with variable name ecma_intl_test_option.